core/admin/mailu/utils.py

```diff
@@ -363,13 +353,12 @@ def parse_key(self, key, app=None, validate=False, now=None):
         if created is None or self._decode(uid) is None or self._decode(sid) is None:
             return None
 
-        # validate creation time when requested or store does not support ttl
-        if validate or not app.session_store.has_ttl:
-            if now is None:
-                now = int(time.time())
-            created = int.from_bytes(created, byteorder='big')
-            if not created < now < created + app.permanent_session_lifetime.total_seconds():
-                return None
+        # validate creation time
+        if now is None:
+            now = int(time.time())
+        created = int.from_bytes(created, byteorder='big')
+        if not created <= now <= created + app.config['PERMANENT_SESSION_LIFETIME']:
```

Comment:
See stack trace. i think app.config['PERMANENT_SESSION_LIFETIME'] must be cast  to a string.
Something like int( app.config['PERMANENT_SESSION_LIFETIME'] )?